### PR TITLE
Account flag overwritten in exo api

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
issue #21 
-A, --account flag is overwritten in some exo api cmd.
help wanted to find another solution

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func Execute(version, commit string) {
 
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&gConfigFilePath, "config", "C", "", "Specify an alternate config file [env EXOSCALE_CONFIG]")
-	RootCmd.PersistentFlags().StringVarP(&gAccountName, "account", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
+	RootCmd.PersistentFlags().StringVarP(&gAccountName, "accountt", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```
Comment (pierre-emmanuelJ) [inline cmd/root.go]: WIP need some proposition

Comment (greut) [reply cmd/root.go]: we remove it and over an easy way to switch between them.

Comment (pierre-emmanuelJ) [reply cmd/root.go]: --ACCOUNT can't be fine ?
```suggestion
	RootCmd.PersistentFlags().StringVarP(&gAccountName, "ACCOUNT", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```

Comment (greut) [reply cmd/root.go]: or longer? `--use-account` ?

Comment (pierre-emmanuelJ) [reply cmd/root.go]: yes can be great !
```suggestion
	RootCmd.PersistentFlags().StringVarP(&gAccountName, "use-account", "A", "", "Account to use in config file [env EXOSCALE_ACCOUNT]")
```

**Pull request merged**